src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,34 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
 import React from 'react';
 import { Loader } from '../Loader';
+import { User } from '../../types/User';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  selectedUser: User,
+  isModalLoaded: boolean,
+  setSelectedTodo: (param: number) => void,
+  setSelectedUser: (param: User) => void,
+  setIsModalLoaded: (param: boolean) => void,
+  todoList: Todo[],
+  selectedTodoId: number,
+  setSelectedTodoId: (param: number) => void,
+};
+  
+export const TodoModal: React.FC<Props> = (
+  {
+    selectedUser, isModalLoaded, setSelectedTodo,
+    setSelectedUser, setIsModalLoaded, todoList,
+    selectedTodoId, setSelectedTodoId,
+  },
+) => {
```

Comment:
```suggestion
export const TodoModal: React.FC<Props> = ({
    selectedUser, isModalLoaded, setSelectedTodo,
    setSelectedUser, setIsModalLoaded, todoList,
    selectedTodoId, setSelectedTodoId,
  }) => {
```
and better for readability to write 1 prop per line